docs/2101UserGuide.md

```diff
@@ -62,50 +62,56 @@ CCACommander Ultra Promax Xtra 9000PLUS is the one-stop app for CCA Heads to man
 
 <hr class="feature-class-separator">
 
-### Member commands
+### Member Commands
+The vitality of a CCA lies in its members. CCACommander allows you to manage your members with the commands below:
 
 #### Create a Member: `createMember`
-Creates a new member with accompanying personal details (name, gender, phone number, email address, home address, tag).
+When you first start CCACommander and want to populate CCACommander with your members, you can do it with just one simple command - `createMember`!
 
-Format: `createMember n/MEMBER_NAME g/GENDER [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Here is how you can use it:<br>
+`createMember n/MEMBER_NAME g/GENDER [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
 
-* Acceptable values for `GENDER`: `Male`, `Female`, `Others`.
-* Acceptable values for `EMAIL`: A string with an email extension (e.g. `@gmail.com`).
+Here is an example command where we create a member **named** "Lim Jun Jie" with **gender** "Male", **phone number** of "83498018", **email** of "limjunjie@gmail.com", home **address** of "19 Kent Ridge Crescent, Singapore 119278" and a "publicity" **tag**:
```

Comment:
Hmm if we include dylan's extra part of `to reflect his role as ....` but not include in some commands will it be inconsistent and wrong?

Reply:
I don't think it is wrong, because it is just about how you want to make it sound friendlier.